### ISSUE TITLE
Request for more dinput buttons

### ISSUE THREAD

**Issue by butterrsoft** [FeralAI/GP2040#50]
The GP2040 encoder is awesome, but i have an improvement in mind. Under dipnut on windows, having 20 buttons + dpad would let ppl run a 2-player MAME cab from a single Pico! I'd personally love to see those 8 blank input pins mapped for that purpose :)

**Comment by matt199x** [comment]
I agree. The more available buttons the better.

**Comment by jbarket** [comment]
Agreed. We’re currently limiting things for console compatibility but there’s no reason we can’t also have a mode that’s just maxed out for PC
